Makefile.am

```diff
@@ -259,7 +259,10 @@ BASE_FILES += \
 	src/CameraServer.h \
 	src/CameraDevice.h \
 	src/CameraDevice_V4L2.cpp \
-	src/CameraDevice_V4L2.h
+	src/CameraDevice_V4L2.h \
```

Comment:
Filename uses mix of CamelCase & snake_case. Please change it CamelCase as it is consistent with existing source files. Same applies to class names as well.

Reply:
Fixed